Library/Homebrew/cmd/test-bot.rb

```diff
@@ -590,6 +590,11 @@ def formula(formula_name)
           if bottle_step.passed? && bottle_step.has_output?
             bottle_filename =
               bottle_step.output.gsub(/.*(\.\/\S+#{bottle_native_regex}).*/m, '\1')
+            bottle_rb_filename = bottle_filename.gsub(/\.(\d+\.)?tar\.gz$/, ".rb")
+            bottle_merge_args = ["--merge", "--write", "--no-commit", bottle_rb_filename]
```

Comment:
Given they are cached locally and these machines all have absurdly fast internet connections I'm not convinced it'll make much of a difference.